lib/datasource/aws-machine-image/index.spec.ts

```diff
@@ -131,11 +131,15 @@ const mock1Image: DescribeImagesResult = {
 
 const mockEmpty: DescribeImagesResult = {};
 
+function mockDescribeImagesCommand(result: DescribeImagesResult): void {
+  ec2Mock.reset();
+  ec2Mock.on(DescribeImagesCommand as never).resolves(result as never);
```

Comment:
Type error too hard for me to decode 🤷‍♂️

Reply:
@viceice are you ok with this?

Reply:
```suggestion
  ec2Mock.on(DescribeImagesCommand).resolves(result);
```

Works for me, no warnings or anything else. 🤔